### PR TITLE
Remove GUI buttons/options for things that no longer have any effect.

### PR DESCRIPTION
**Pull request by jannistsiroyannis**

### CODE REVIEW

File: addjob.jsp [hunk 1]
```diff
@@ -14,7 +14,7 @@
                 </tr>
                 <tr>
                   <td colspan="2">
-                  <font color="#ff0000">OBS! Lägg inte ett startdatum som är äldre än två år. Sökningen blir då för tung och kommer inte att fungera. Använd "fullständigt uttag" i vänstermenun om det är det ni vill ha.</font></td></tr>
+                  <font color="#ff0000">OBS! Lägg inte in ett interval som är större än 6 månader. Sökningen blir då för tung och kommer inte att fungera. Använd "fullständigt uttag" i vänstermenun om det är det ni vill ha.</font></td></tr>
```
Comment (olovy) [inline addjob.jsp]: typo interval/intervall

File: addjob.jsp [hunk 2]
```diff
@@ -165,43 +165,9 @@
...
-                    <td colspan="2" class="default"><b>&nbsp;e-post</b></td>
-                </tr>
-                <tr>
-                    <td nowrap class="default">&nbsp;&nbsp;statusrapport till:</td>
-                    <td><input type="text" size="30" name="contact" value="<% out.print(properties.getProperty("contact", "")); %>"/></td>
-                </tr>
-                <tr>
-                    <td nowrap class="default">&nbsp;&nbsp;felrapport till:</td>
-                    <td><input type="text" size="30" name="errcontact" value="<% out.print(properties.getProperty("errcontact", "")); %>"/></td>
-                </tr>
```
Comment (olovy) [inline addjob.jsp:204]: These exist in the data.
Not needed anymore?

Comment (olovy) [reply addjob.jsp:204]: Should be handled in bibdb I guess

**Review by olovy — APPROVED** [review]
:ship:

File: addjob.jsp (hunk 1 above)
Comment (klngwll) [reply addjob.jsp]: och vänstermenu/meny

Comment (jannistsiroyannis) [reply addjob.jsp]: Japp! Ska fixa!

File: addjob.jsp (hunk 2 above)
Comment (jannistsiroyannis) [reply addjob.jsp:204]: All of the things this PR removes exist in the data (in the profiles), but we no longer use them.

**Pull request merged**